README.md

```diff
@@ -1,41 +1,112 @@
+
+##### Price Changes (iOS Only)
 
-## iOS
+The `date` field is the date that the price was effective on that product. For example, if a product, called `premium_upgrade`, was created on October 19, 2021 and the price was raised from 3.99 USD to 6.99 USD on December 13, 2021, the product price map would have two rows for the product:
 
-`app_user_id` 
+```
+product_id,price,currency,introductory_price,date,duration,introductory_price_duration
+premium_upgrade,3.99,USD,0.00,2021-10-19,P6M,P1M
+premium_upgrade,6.99,USD,0.00,2021-12-13,P6M,P1M
+...
+```
 
-`receipt` (the raw receipt file)
+> 💡 At least one row for USD is required. If you don't provide a price for a specific currency, RevenueCat will always default to USD for purchases made in that currency.
 
-`product_id`
+### Android
 
-`price`
+> 💡 Check out the [sample file](iOS/ios_product_price_map_sample.csv) for a complete example of a product price map CSV file for iOS.
 
-`introductory_price` (if not a free trial)
+> ⚠️ We need to know your [Google Play API quota](https://developers.google.com/android-publisher/quotas) and approximately how many remaining requests per day you have so we don't exceed your quota.
 
-`currency`
+`app_user_id` The user identifier to associate with the token.
 
-*If you don't have prices and currency with the transactions we can take a separate file with a mapping of product_id, price, currency as well. If there were any known price changes, those should also be included here.
+`product_id` The product ID associated with the token.
 
-> Note: The entire iOS receipt file must be used for imports. The latest_receipt_info is not supported as it's only a subset of the entire receipt.
+`token` The purchase token from the device.*
 
-## Android:
+***Note that Android receipts that expired more than 90 days ago can't be imported.** Additionally, only the current status can be retrieved from each Android token. This means that Android charts for historical data won't be accurate.
 
-`app_user_id`
+### Stripe
 
-`product_id`
+> 💡 Check out the [sample file](iOS/ios_product_price_map_sample.csv) for a complete example of a product price map CSV file for iOS.
 
-`token`
+`app_user_id` The user identifier to associate with the subscription.
 
-Important: We need to know your API quota with Google and approximately how many remaining requests per day you have so we don't exceed your quota.
+`subscription_token` (e.g. `sub_xxxxxx`) The subscription token.*
 
-*Note that Android receipts that expired more than 90 days ago can't be imported, and only the current status can be retrieved from each Android receipt. This means that Android charts for historical data won't be accurate.
+***Note that Stripe subscription objects contain only the current status of the subscription.** This means that Stripe charts for historical data won't be accurate.
```

Comment:
This callout is also new to bring attention to the fact that Stripe doesn't store complete subscription history in the token

Reply:
Is the Google purchase token validation period 60 days or 90 days? I told TuneIn today it was 60 days 😅

Doesn't seem clear why a product price map would be required on Android since price isn't a required field in the receipt?

Reply:
[60 days is correct](https://developer.android.com/google/play/billing/subscriptions#lifecycle), that must be an old mistake. Although if you have a grace period it might "extend" it to 90 days, maybe that's where that came from?

![image](https://user-images.githubusercontent.com/6395378/147605623-65ff7296-e5ea-4ed4-8e14-4398f50e7a59.png)

And lol, copy-paste error, neither Android nor Stripe require a product price map

Reply:
60 days fix [here](https://github.com/RevenueCat-Samples/import-csv-samples/pull/4/commits/fe3a2b034d7b27f65227477f3176e1d93b237353)

Links fixed [here](https://github.com/RevenueCat-Samples/import-csv-samples/pull/4/commits/75e2ee97140a4e9e6593cf27092eaf49be5c0951)